jpa/subsystem/src/main/java/org/jboss/as/jpa/processor/JPADependencyProcessor.java

```diff
@@ -24,8 +24,10 @@
 
```

Comment:
Change for sub-deployment components to only depend on persistence units from the same sub-deployment when handling `initialize-in-order`.